buildlib/docker_tests.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
```

Comment:
this file seems a duplication of tests.yml. what is the difference? can we reuse it and maybe just tweak some params?